packages/web-components/src/components/rux-clock/rux-clock.scss

```diff
@@ -12,13 +12,18 @@
     /**
     * @prop --clock-border-color: the border color for the clock
     */
-    --clock-border-color: var(--color-global-snowflakes-dark-surface);
+    --clock-border-color: var(--color-surface);
 
     /**
     * @prop --clock-label-color: the label color for the clock
     */
     --clock-label-color: var(--color-global-basics-white);
 
+    /**
+    * @prop --clock-label-background-color: the background color for clocks date, time, aos, and los labels.
+    */
+    --clock-label-background-color: var(--color-gsb-global-status-bar);
```

Comment:
nix this and just use the token. we're going to be removing a lot of these css custom props now that we have parts in 7.0